Packs/ServiceNow/Integrations/ServiceNowv2/Layouts/layout-details-ServiceNowTicket-V2.json

```diff
@@ -0,0 +1,701 @@
+{
+	"TypeName": "ServiceNowTicket",
```

Comment:
this should be `"toVersion": "5.9.9"` and need to add another layout to be `"fromVersion": "6.0.0"` according to new structure